index.html

```diff
@@ -75,5 +75,251 @@ <h4>LETS CONNECT</h4>
                     </div>
               </div>
        </section> 
+       <section class="grid-container">
+             <div class="project-item">
+                <div class="project-image">
+                  <img src="./assets/first-image.jpg" alt="first project image"/>
```

Comment:
- [ ] Your image has not the same proportion as in the Figma template.

FIGMA:
![Captura de Pantalla 2022-09-23 a la(s) 8 08 50 a  m](https://user-images.githubusercontent.com/86022690/191967513-84a049d0-a55b-44f2-81ff-795e8d7a1fb5.png)

YOURS:
![Captura de Pantalla 2022-09-23 a la(s) 8 09 02 a  m](https://user-images.githubusercontent.com/86022690/191967534-4593bc68-0e91-47f5-b33e-acb841b511b0.png)

As you can see the image is being stretched because of this. 👍